view/widget/nbp_results_view_frame.py

```
@@ -0,0 +1,10 @@
+
+from service.chart.chart_component import ChartComponent
+from view.model.viewframe import ViewFrame
+
+
+class NbpResultsView(ViewFrame):
+    def __init__(self, parent) -> None:
```

Ruff

Missing type annotation for `self` in method

(ANN101)